Cue/app/tabs/library/LibraryHome.js

```diff
@@ -66,16 +66,18 @@ class LibraryHome extends React.Component {
 
   constructor(props: Props) {
     super(props)
-
+
+    let refreshing = this.props.decks ? false : true;

```

Comment:
Oh, I removed it since a `syncDeck` action would trigger it and stop the refreshing icon before `loadLibrary` was finished